HUBER/iocBoot/iocHUBER-IOC-01/st-motor.cmd

```diff
@@ -5,12 +5,24 @@ set_pass0_restoreFile("$(IOCNAME)_built_settings.sav")
 set_pass1_restoreFile("$(IOCNAME)_built_settings.sav")
 
 $(IFDEVSIM) drvAsynIPPortConfigure("Huber$(MTRCTRL)", "localhost:$(EMULATOR_PORT)")
-$(IFNOTSIM) drvAsynIPPortConfigure("Huber$(MTRCTRL)", "$(HUBERADDR01)")
+$(IFNOTDEVSIM) $(IFNOTRECSIM) drvAsynSerialPortConfigure("Huber$(MTRCTRL)", "$(PORT)", 0, 0, 0)
+$(IFNOTDEVSIM) $(IFNOTRECSIM) asynSetOption("Huber$(MTRCTRL)", -1, "parity", "$(PARITY=None)")
+$(IFNOTDEVSIM) $(IFNOTRECSIM) asynSetOption("Huber$(MTRCTRL)", -1, "baud", "$(BAUD=19200)")
+$(IFNOTDEVSIM) $(IFNOTRECSIM) asynSetOption("Huber$(MTRCTRL)", -1, "bits", "$(BITS=8)")
+
+$(IFNOTDEVSIM) $(IFNOTRECSIM) asynSetOption("Huber$(MTRCTRL)", -1, "stop", "$(STOP=1)")
+## Hardware flow control off
+$(IFNOTDEVSIM) $(IFNOTRECSIM) asynSetOption("Huber$(MTRCTRL)", 0, "clocal", "Y")
+$(IFNOTDEVSIM) $(IFNOTRECSIM) asynSetOption("Huber$(MTRCTRL)",0,"crtscts","N")
+## Software flow control off
+$(IFNOTDEVSIM) $(IFNOTRECSIM) asynSetOption("Huber$(MTRCTRL)",0,"ixon","N")
+$(IFNOTDEVSIM) $(IFNOTRECSIM) asynSetOption("Huber$(MTRCTRL)",0,"ixoff","N")
```

Comment:
possibly nitpicking, but as these are serial ports they are probably better placed in st-common.cmd as they aren't motion control related and rather higher-level comms

Reply:
we can leave these in here as they rely on $(mtrctrl)